commons/all/src/main/java/org/infinispan/commons/configuration/attributes/IdentityAttributeCopier.java

```diff
@@ -12,7 +12,7 @@
 public class IdentityAttributeCopier<T> implements AttributeCopier<T> {
    public static final AttributeCopier<Object> INSTANCE = new IdentityAttributeCopier<>();
```

Comment:
Why are we moving away from using `INSTANCE` variables in the copier implementations?

Reply:
Because I preferred creating new instances (there aren't many) instead of having lots of casts because you cannot really have a  static with generic wildcards

Reply:
Sounds good to me. Just make sure that all of the unused `INSTANCE` vars are removed.